cirq/experiments/google_v2_supremacy_circuit.py

```diff
@@ -105,6 +105,41 @@ def google_v2_supremacy_circuit_grid(n_rows: int, n_cols: int,
     return google_v2_supremacy_circuit(qubits, cz_depth, seed)
 
 
+def google_v2_supremacy_circuit_bristlecone(n_rows: int,
+                                            cz_depth: int, seed: int
+                                            ) -> circuits.Circuit:
+    """
+    Generates Google Random Circuits v2 in Bristlecone.
+    See also https://arxiv.org/abs/1807.10749
+
+    Args:
+        n_rows: number of rows in a Bristlecone lattice.
+          Note that we do not include single qubit corners.
+        cz_depth: number of layers with CZ gates.
+        seed: seed for the random instance.
+
+    Returns:
+        A circuit with given size and seed.
+    """
+    def get_qubits(n_rows):
+        half_rows = n_rows // 2
```

Comment:
```python
min_row = 5 - n_rows // 2
max_row = min_row + n_rows - 1
dev = cirq.google.Bristlecone
qubits = [q for q in dev.qubits
          if min_row <= q.row <= max_row and len(dev.neighbors_of(q)) > 1]
```

Reply:
![sub-brislecone](https://user-images.githubusercontent.com/10459129/47337068-33824e80-d647-11e8-8f58-89e4d71fc810.jpg)

The qubits I want for n_rows = 9 are inside the blue shape, and for n_rows = 10 are in the inside the red shape.

Reply:
Oh, I've misunderstood the rule. Could you label the other sizes as well? You might want to include such diagrams in the code as docs. Is it always nearly a square?